dedibox-network/dns/concepts.mdx

```diff
@@ -5,13 +5,6 @@ meta:
 content:
   h1: Concepts
   paragraph: This page explains all the concepts related to DNS
-tags: advanced settings
```

Comment:
did you mean to delete the tags and categories?

Reply:
no need for them here (only taken into consideration for tutorials)